src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,52 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService = (ManufacturerService) injector
+                .getInstance(ManufacturerService.class);
+        DriverService driverService = (DriverService) injector
+                .getInstance(DriverService.class);
+        Manufacturer teslaManufacturer = new Manufacturer("Tesla", "USA");
+        teslaManufacturer = manufacturerService.create(teslaManufacturer);
```

Comment:
no need to reset the value of the variable here
the object already got the id anyway